### ISSUE TITLE
Test environment?

### ISSUE THREAD

**Issue by AymericG** [brainblocks/brainblocks#30]
It would be great if I could test the payments without really needing to send actual coins. 

For example, you could have a different test environment (url) for your api, and some amounts will always return the same response:

1.23 raiblocks: Always returns an ERROR

**Comment by bluepnume** [comment]
Good idea 👍 I'm not sure if there's any kind of raiblocks/nano test network? But either way, having a facility for negative testing is a good idea.
